### PR TITLE
Size command

### PR DESCRIPTION
**Pull request by alanj853**

### CODE REVIEW

File: lib/ftp/bifrost.ex [hunk 1]
```diff
@@ -454,6 +454,30 @@ defmodule Ftp.Bifrost do
...
+            root_dir: root_dir,
+            current_directory: current_directory
+          } = state,
+          path
+        ) do
+      working_path = determine_path(root_dir, current_directory, path)
+  
+      case encode_file_info(permissions, working_path) do
+        nil -> {"-1", state}
+        info -> {Tuple.to_list(info) |> Enum.at(6) |> to_string(), state}
```
Comment (adkron) [inline lib/ftp/bifrost.ex]: `{info |> elem(6) |> to_string(), state}` will be faster.

File: lib/ftp/bifrost.ex [hunk 2]
```diff
@@ -454,6 +454,30 @@ defmodule Ftp.Bifrost do
...
+            permissions: permissions,
+            root_dir: root_dir,
+            current_directory: current_directory
+          } = state,
+          path
+        ) do
+      working_path = determine_path(root_dir, current_directory, path)
+  
+      case encode_file_info(permissions, working_path) do
+        nil -> {"-1", state}
```
Comment (adkron) [inline lib/ftp/bifrost.ex]: Instead of an esoteric size of -1 could we use something more descriptive, `:unknown` seems better to me.

Comment (adkron) [reply lib/ftp/bifrost.ex]: Or if we know the reason we could put that there. I also see in other plaves we just return `:error` and no state, I think.

**Comment by alanj853** [comment]
Tested this with the HMI, all tests pass. Merging to development.

**Pull request merged**